neutrons/SNAPRed
# Update CalibrantSample Classes

**Description of work**

Updated CalibrantSample classes to match SetSample
 - Geometry and Material classes had members updated to match what is being used in mantid's SetSample() function

**To test:**

On the analysis cluster, run mantid from SNAPRed and run a script that is able to call mantid's SetSample() using Geometry and Material objects created from a CalibrantSamples() object. SetSample() should not fail when called.


Link to EWM item
[Story 2436](https://ornlrse.clm.ibmcloud.com/ccm/web/projects/Neutron%20Data%20Project%20%28Change%20Management%29#action=com.ibm.team.workitem.viewWorkItem&id=2436)
